src/router/controller/params/decorators.ts

```diff
@@ -55,7 +79,32 @@ export const Body = (prop?: string) =>
 		if (!body) {
 			return null;
 		}
-		return prop ? body[prop] : body;
+
+		// Extract Class type of Parameter with @Body
+		const { parameterIndex } = opts;
+		const params: Constructor[] = MetadataHelper.getMetadata(
+			'design:paramtypes',
+			opts.target,
+			opts.propertyKey,
+		);
+
+		// Make the validation of body
+		if (isArray(params)) {
+			const validation = validateObject(body, params[parameterIndex]);
```

Comment:
`validation` maybe be named `errors` for clarity, what do you think ?

Reply:
I totally agree!